addons/website_sale/models/website.py

```diff
@@ -205,8 +204,8 @@ def sale_get_order(self, force_create=False, code=None, update_pricelist=False,
         if not sale_order_id:
             last_order = partner.last_website_so_id
             available_pricelists = self.get_pricelist_available()
-            # Do not reload the cart of this user last visit if the cart is no longer draft or uses a pricelist no longer available.
-            sale_order_id = last_order.state == 'draft' and last_order.pricelist_id in available_pricelists and last_order.id
```

Comment:
Imho this draft condition needs to remain since after a checkout the call to `sale_get_order` need to returns a new SO if the last one is in sent
And without this draft condition it will return the last sent and wont create a new one
Then after a checkout you will remain forever with the sent one